<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompilerParamWindow</class>
 <widget class="QMainWindow" name="CompilerParamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_overallOptions">
      <property name="text">
       <string>Options générales</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_overallOptions"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_overallOptions">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_languageCOptions">
      <property name="text">
       <string>Options de langage C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_languageCxxOptions">
      <property name="text">
       <string>Options de langage C++</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_languageCOptions"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_languageCOptions">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_languageCxxOptions"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_languageCxxOptions">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
